src/client/thingtype.cpp

```diff
@@ -520,7 +524,50 @@ void ThingType::unserialize(const uint16_t clientId, const ThingCategory categor
 
         totalSpritesCount += totalSprites;
     }
-
+    if (sizes.size() > 1) {
+        bool hasDifferentSizes = false;
+        const Size& firstSize = sizes[0];
+        for (size_t i = 1; i < sizes.size(); ++i) {
+            if (sizes[i] != firstSize) {
+                hasDifferentSizes = true;
+                break;
+            }
+        }
+        if (hasDifferentSizes) {
+            g_logger.warning("[{}] g_game.getLocalPlayer():setOutfit({{type={}}})", __FUNCTION__, m_id);
```

Comment:
Future note: remove this before merging.

```suggestion
            g_logger.warning("[{}] g_game.getLocalPlayer():setOutfit({{type={}}})", __FUNCTION__, m_id);
```